File: 1_iniciante/1161/1161.py
"""
BEE 1161 - Soma de Fatoriais - Nível 5 - Matemática
"""


def fatorial(N):
    if N == 0 or N == 1:
        return 1
    return N * fatorial(N - 1)


while True:
    try:
        numbers = input().split()
        print(fatorial(int(numbers[0])) + fatorial(int(numbers[1])))
    except EOFError:  # eof
        break

File: 1_iniciante/1161/1161.cpp
// 
// BEE 1161 - Soma de Fatoriais - Nível 5 - Matemática
// 
#include <iostream>

using namespace std;

long long int Fatorial(int N)
{
    if (N == 0)
        return 1;
    return N * Fatorial(N - 1);
}

int main()
{
    int M, N;

    while (cin >> M >> N)
    {
        cout << Fatorial(M) + Fatorial(N) << endl;
    }

    return 0;
}